modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_user_registration/create_account_user.jsp

```diff
@@ -156,6 +156,7 @@ portletDisplay.setURLBack(backURL);
 			</div>
 
 			<div class="form-group">
+			<c:if test="<%= PrefsPropsUtil.getBoolean(company.getCompanyId(), PropsKeys.LOGIN_CREATE_ACCOUNT_ALLOW_CUSTOM_PASSWORD, PropsValues.LOGIN_CREATE_ACCOUNT_ALLOW_CUSTOM_PASSWORD) %>">
```

Comment:
Indentation is not aligned, i suppose we should run `gw formatSource` though. Should we?